src/PlatformSwitch/PlatformSwitch.stories.tsx

```diff
@@ -0,0 +1,56 @@
+import { ComponentMeta, ComponentStory } from '@storybook/react';
+
+import React from 'react';
+
+import { PRIMARY_COLOR, SECONDARY_COLOR } from '../theme';
+import PlatformSwitch from './PlatformSwitch';
+import { Platform } from './hooks';
+
+export default {
+  title: 'Common/PlatformSwitch',
+  component: PlatformSwitch,
+} as ComponentMeta<typeof PlatformSwitch>;
+
+const Template: ComponentStory<typeof PlatformSwitch> = (args) => (
+  <PlatformSwitch {...args} />
+);
+
+export const Light = Template.bind({});
+Light.args = {
+  color: PRIMARY_COLOR,
+  accentColor: SECONDARY_COLOR,
+  selected: Platform.Builder,
+};
+
+export const Dark = Template.bind({});
+Dark.args = {
+  color: SECONDARY_COLOR,
+  accentColor: PRIMARY_COLOR,
+  selected: Platform.Builder,
+};
+Dark.parameters = {
+  backgrounds: { default: 'dark' },
+};
+
+export const Disabled = Template.bind({});
+Disabled.args = {
+  color: PRIMARY_COLOR,
+  accentColor: SECONDARY_COLOR,
+  selected: Platform.Builder,
+  platformsProps: {
+    Builder: {
+      onClick: () => alert('You clicked on Builder'),
+    },
+    Player: {
+      onClick: () => alert('You clicked on Player'),
+    },
+    Library: {
+      onClick: () => alert('You clicked on Library'),
+    },
+    Analytics: {
+      disabled: true,
+      // this onClick should not trigger when disabled: true
+      onClick: () => alert('You clicked on Analytics'),
+    },
+  },
+};
```

Comment:
You could add tests, if you really want (check your buttons are clickable) https://storybook.js.org/docs/react/writing-tests/interaction-testing

Reply:
I think the interactions are simple enough, given the UI component only exposes `onClick`s. The integrations tests will make more sense IMO.